src/libraries/Microsoft.PowerFx.Core/Parser/ParseResult.cs

```diff
@@ -58,33 +58,32 @@ public class ParseResult : IOperationStatus
         // All the spans in the tokens are relative to this. 
         public string Text { get; }
 
+        public CultureInfo ParseCulture { get; } 
```

Comment:
>ParseCulture [](http://example.com/codeflow?start=27&length=12)

This should be same as this.Options.Cutlure

Reply:
Good point, I'll fix